site/content/docs/main/restore-reference.md

```diff
@@ -142,3 +189,76 @@ data:
   # node name and the value is the new node name.
   <old-node-name>: <new-node-name>
 ```
+
+## Restoring into a different namespace
+
+Velero can restore resources into a different namespace than the one they were backed up from. To do this, use the `--namespace-mappings` flag:
+
+```bash
+velero restore create <RESTORE_NAME> \
+  --from-backup <BACKUP_NAME> \
+  --namespace-mappings old-ns-1:new-ns-1,old-ns-2:new-ns-2
+```
+
+For example, A Persistent Volume object has a reference to the Persistent Volume Claim’s namespace in the field `Spec.ClaimRef.Namespace`. If you specify that Velero should remap the target namespace during the restore, Velero will change the  `Spec.ClaimRef.Namespace` field on the PV object from `old-ns-1` to `new-ns-1`.
+
+## Restore existing resource policy
+
+By default, Velero is configured to be non-destructive during a restore. This means that it will never overwrite data that already exits in your cluster. When Velero attempts to create a resource during a restore, the resource being restored is compared to the existing resources on the target cluster by the Kubernetes API Server. If the resource already exists in the target cluster, Velero skips restoring the current resource and moves onto the next resource to restore, without making any changes to the target cluster.
+
+An exception to the default restore policy is ServiceAccounts. When restoring a ServiceAccount that already exists on the target cluster, Velero will attempt to merge the fields of the ServiceAccount from the backup into the existing ServiceAccount. Secrets and ImagePullSecrets are appended from the backed-up ServiceAccount. Velero adds any non-existing labels and annotations from the backed-up ServiceAccount to the existing resource, leaving the existing labels and annotations in place.
+
+You can change this policy for a restore by using the `--existing-resource-policy` restore flag. The available options are `none` (default) and `update`. If you choose to `update` existing resources during a restore, Velero will attempt to update an existing to match the resource being restored.
```

Comment:
Missing a word "Velero will attempt to update an existing to match the resource being restored"